src/wp-includes/class-wp-customize-manager.php

```diff
@@ -1365,12 +1365,7 @@ function import_theme_starter_content( $starter_content = array() ) {
 						)
 					);
 
-					// In PHP < 5.6 filesize() returns 0 for the temp files unless we clear the file status cache.
-					// Technically, PHP < 5.6.0 || < 5.5.13 || < 5.4.29 but no need to be so targeted.
-					// See https://bugs.php.net/bug.php?id=65701
-					if ( version_compare( PHP_VERSION, '5.6', '<' ) ) {
-						clearstatcache();
-					}
+					clearstatcache();
```

Comment:
> In PHP < 5.6 filesize() returns 0 for the temp files unless we clear the file status cache.

I don't know what this means, and I don't care.  Being able to get rid of stuff like this is why we upgrade software versions, folks.  :heavy_check_mark:

Reply:
Ah, and actually we should get rid of the `clearstatcache()` call too, because we are never `< 5.6` any more.